transcribe/scribe_modules/lspci.py

```diff
@@ -0,0 +1,34 @@
+from . import ScribeModuleBaseClass
+
+import sys
+
+from .lib.util import validate_length
+
+
+class Lspci(ScribeModuleBaseClass):
+
+    def __init__(self, input_dict=None, module_name=None, host_name=None,
+                 input_type=None, scribe_uuid=None):
+        ScribeModuleBaseClass.__init__(self, module_name=module_name,
+                                       input_dict=input_dict,
+                                       host_name=host_name,
+                                       input_type=input_type,
+                                       scribe_uuid=scribe_uuid)
+
+    def parse(self):
+        lspci_data = self._input_dict
+        lspci_lines = lspci_data.split('\n\n')
+
+        validate_length(len(lspci_lines), self.module)
+
+        # lspci_lines contains a list of slots and its information.
+        # We are yielding a dict of these individual slots.
+        for i in range(len(lspci_lines)):
+            slot_data = lspci_lines[i].split('\n')
+            # sample - ['Slot:\t00:00.0', 'Class:\tHost bridge', 'Vendor:\tIntel Corporation' ..... ]
+            self._dict["value"] = {}
```

Comment:
move this to line 31, so that we make sure we're not reusing any key,value pairs from previous slot data

Reply:
The values are already reset for each slot data in the first for loop. The second loop is for data in each slot, so we will need to move this.